mnemosyne/libmnemosyne/configuration.py

```diff
@@ -460,6 +460,9 @@ def default_language(self):
         # use the default OS language settings is possible
         # getdefaultlocale is OS independent and works on all platforms
         system_language, _ = getdefaultlocale()
+        # if no default locale can be found, default to English
+        if system_language is None:
+            system_language = 'en_us'
```

Comment:
can you clarify why this is a necessary part of this change (or if it's just another bonus change you included)?

Reply:
This is a bonus fix, otherwise Mnemosyne would crash on my system.